rocky/rocky/views/mixins.py

```diff
@@ -259,20 +259,19 @@ def __getitem__(self, key: Union[int, slice]) -> List[HydratedFinding]:
         raise NotImplementedError("FindingList only supports slicing")
 
 
+_EXCLUDED_OOI_TYPES = ("Finding", "FindingType")
```

Comment:
Import this from `ooi_form.py`

Reply:
https://github.com/minvws/nl-kat-coordination/issues/2108